app/globals.css

```
@@ -6,8 +6,8 @@
 @theme inline {
   --color-background: var(--background);
   --color-foreground: var(--foreground);
-  --font-sans: DM Sans, sans-serif;
-  --font-mono: IBM Plex Mono, monospace;
+  --font-sans: Inter, system-ui, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)